examples/create-setters-simple/README.md

```diff
@@ -83,7 +85,7 @@ $ kpt fn render create-setters-simple
 
 ### Expected result
```

Comment:
`create-setters` will be most commonly used in imperative way, right ? we should probably show only that in the examples. @phanimarupaka ? It will also avoid the problems where publishers would publish the packages with `create-setters` in the package pipeline.